angular 4 - missing @angular/tsc-wrapped dependency and wrong version of this package

> Please provide us with the following information:
> ---------------------------------------------------------------

### OS?

OS X Sierra 10.12.3

### Versions.

```
@angular/cli: 1.0.0-beta.31
node: 7.4.0
os: darwin x64
@angular/common: 4.0.0-beta.7
@angular/compiler: 4.0.0-beta.7
@angular/core: 4.0.0-beta.7
@angular/forms: 4.0.0-beta.7
@angular/http: 4.0.0-beta.7
@angular/platform-browser: 4.0.0-beta.7
@angular/platform-browser-dynamic: 4.0.0-beta.7
@angular/router: 4.0.0-beta.7
@angular/cli: 1.0.0-beta.31
@angular/compiler-cli: 4.0.0-beta.7
```

### Repro steps.

Init new project with `-ng4` option.

```
$ ng init -ng4
```

### The log given by the failure.
> Normally this include a stack trace and some more information.

```
$ ng init -ng4
...
$ npm list
...
npm ERR! peer dep missing: @angular/tsc-wrapped@^0.5.0, required by @ngtools/webpack@1.2.9
```

### Mention any other details that might be useful.

Package `@angular/tsc-wrapped` has 4.0.0-beta.7 version.

--- a/packages/@ngtools/webpack/package.json
+++ b/packages/@ngtools/webpack/package.json
@@ -34,7 +34,7 @@
     "@angular/compiler": "^2.3.1 || >=4.0.0-beta <5.0.0",
     "@angular/compiler-cli": "^2.3.1 || >=4.0.0-beta <5.0.0",
     "@angular/core": "^2.3.1 || >=4.0.0-beta <5.0.0",
-    "@angular/tsc-wrapped": "^0.5.0",
+    "@angular/tsc-wrapped": "^0.5.0 || >=4.0.0-beta <5.0.0",
     "typescript": "^2.0.2",
     "webpack": "^2.2.0"
   }

